tapdb/supply_tree.go

```diff
@@ -596,3 +596,111 @@ func (s *SupplyTreeStore) ApplySupplyUpdates(ctx context.Context,
 	return finalRoot, nil
 }
 
+// SupplyUpdate is a struct that holds a supply update event and its block
+// height.
+type SupplyUpdate struct {
+	supplycommit.SupplyUpdateEvent
+	BlockHeight uint32
+}
+
```

Comment:
`supplycommit.SupplyUpdateEvent` already includes 
```
// BlockHeight returns the block height of the update.
BlockHeight() uint32
```

so I don't think we need the separate `BlockHeight uint32` field in `SupplyUpdate`.